docs/DeployConfigurations.rst

```diff
@@ -97,7 +135,40 @@ Benefit: no performance impact on Nextcloud host.
 		class ExApp2 python
 		class ExApp3 python
 
-In this case, the AppAPI (Nextcloud) uses ``port`` to interact with Docker.
+In this case, the AppAPI (Nextcloud) uses ``port`` to interact with remote Docker, which also could be a Docker Socket Proxy exposed with TLS.
+
+.. mermaid::
+
+	stateDiagram-v2
+		classDef docker fill: #1f97ee, color: transparent, font-size: 34px, stroke: #364c53, stroke-width: 1px, background: url(https://raw.githubusercontent.com/cloud-py-api/app_api/main/docs/img/docker.png) no-repeat center center / contain
+		classDef nextcloud fill: #006aa3, color: transparent, font-size: 34px, stroke: #045987, stroke-width: 1px, background: url(https://raw.githubusercontent.com/cloud-py-api/app_api/main/docs/img/nextcloud.svg) no-repeat center center / contain
+		classDef python fill: #1e415f, color: white, stroke: #364c53, stroke-width: 1px
+
+		Direction LR
+
+			Host1 --> Host2 : by port
+
+		state Host1 {
+			Nextcloud
+		}
+
+		state Host2 {
+			[*] --> DockerSocketProxy : by port
+			Daemon --> Containers
+
+			state Containers {
+				[*] --> DockerSocketProxy : /var/run/docker.sock
+				DockerSocketProxy --> ExApp1
+				DockerSocketProxy --> ExApp2
+				DockerSocketProxy --> ExApp3
+			}
+		}
+
+		class Nextcloud nextcloud
+		class Daemon docker
+		class ExApp1 python
+		class ExApp2 python
+		class ExApp3 python
```

Comment:
cannot judge the changes above, (no preview available)

Reply:
![Screenshot_5](https://github.com/cloud-py-api/app_api/assets/29692256/44077341-7a00-47d8-b746-ca0e924d1e7f)